README.md

```diff
@@ -38,14 +40,37 @@ or if you installed as an Ember CLI command run
 
 `ember update`
 
-This will update your app or addon to the latest Ember CLI version. It does this by fetching the latest version and comparing it to your project's Ember CLI version. It then applies a diff of the changes from the latest version to your project. It will only modify the files if there are changes between your project's version and the latest version, and it will only change the section necessary, not the entire file.
 
-This is different from the existing `ember init` command. That command tries to reset your project back to a brand new project. It removes all your changes and additions.
```

Comment:
We should leave this, because people have asked how this project is different than `ember init`.

Reply:
Added bac as a bullet point under the first use case